snuba/datasets/entities/factory.py

```diff
@@ -35,7 +34,6 @@ def get_entity(name: EntityKey) -> Entity:
 
     entity_factories: MutableMapping[EntityKey, Callable[[], Entity]] = {
         EntityKey.DISCOVER: DiscoverEntity,
-        EntityKey.ERRORS: ErrorsEntity,
```

Comment:
This seems to almost be going to back how Discover currently works. In the sense that users will query for the "events" entity, whether they want errors or all events in general. It also seems like we open ourselves up to more impossible queries, like `"MATCH (events) SELECT exception_stacks.type WHERE type = 'default'..."`

Reply:
I'm not sure I follow exactly. Why is the query you posted impossible? The events entity only returns non transaction events. If they want everything, they need to use the Discover entity.

Reply:
`exception_stacks.type` should not be on the discover entity as it exists only on errors.

Reply:
I think I'm confusing myself. I thought that the errors entity was meant to be particular to errors, in the way transactions are particular to transactions. Since there's just one events table for everything exception transactions, this makes more sense.